src/api/protectedApiClient.ts

```diff
@@ -40,6 +44,16 @@ const deactivateAccount = (): Promise<void> => {
     .catch((err) => err);
```

Comment:
Not related to my PR but why are we catching errors here? It will be impossible to catch them in the thunk so we will get a "Completed" action with an error response

Reply:
a "Completed" action with an error response does not sound like a good idea. I would remove the error catching if it is not possible to catch here

Reply:
I'm pretty sure after catching, we still return a rejected promise to the thunk